<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>357</width>
    <height>201</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Resistencias</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QPushButton" name="btnCodificador">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>30</y>
      <width>291</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Codificador</string>
    </property>
   </widget>
   <widget class="QPushButton" name="btnDecodificador">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>90</y>
      <width>291</width>
      <height>40</height>
     </rect>
    </property>
    <property name="text">
     <string>Decodificador</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>357</width>
     <height>36</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuModo">
    <property name="title">
     <string>Modo</string>
    </property>
    <addaction name="actionCodificador"/>
    <addaction name="actionDecodificador"/>
   </widget>
   <widget class="QMenu" name="menuAcerca_de">
    <property name="title">
     <string>Acerca de ...</string>
    </property>
   </widget>
   <addaction name="menuModo"/>
   <addaction name="menuAcerca_de"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionCodificador">
   <property name="text">
    <string>Codificador</string>
   </property>
  </action>
  <action name="actionDecodificador">
   <property name="text">
    <string>Decodificador</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
